.gitignore

```diff
@@ -3,4 +3,4 @@ node_modules
 .cache
 *.log
 
-dist/
+# dist/
```

Comment:
Do you want to leave it as a comment?

Reply:
yes, when the action is run it uses files from `dist` folder and therefore it has to be committed